projects/server/middleware/verifyToken.js

```diff
@@ -1,27 +1,29 @@
 const { validateToken } = require("../lib/jwt");
 
 module.exports = {
-	tokenVerify: (req, res, next) => {
-		let { token } = req.headers;
+  tokenVerify: (req, res, next) => {
+    let { token } = req.headers;
 
-		if (!token) {
-			return res.status(401).send({
-				isError: true,
-				message: "Token not found",
-				data: null,
-			});
-		}
-		try {
-			const validateTokenResult = validateToken(token)
-			req.uid = validateTokenResult
+    if (!token) {
+      return res.status(401).send({
+        isError: true,
+        message: "Token not found",
+        data: null,
+      });
+    }
 
-			next()
-		} catch (error) {
-			res.status(401).send({
-				isError: true,
-				message: "Invalid Token",
-				data: null,
-			});
-		}
-	},
+    try {
+      const validateTokenResult = validateToken(token);
+      // console.log(validateTokenResult)
+      req.uid = validateTokenResult;
```

Comment:
prefer to store in response.data . then put to either body or query when running cartController